start.js

```diff
@@ -65,7 +65,7 @@ function start (args, cb) {
   })
 
   if (opts['watch']) {
-    return watch(args)
+    return watch(args, parsedArgs(args))
```

Comment:
This will parse all the options again. You can just use the already parsed `opts` from above.

Also you will need to rebase these on top of #129 :(